README.md

```diff
@@ -1 +1,56 @@
 # Canopy Frontend RFC
+The Canopy Frontend RFC (Request for Comments) is a git repository where the technologies and processes that apply to all
+frontend squads at Canopy are described.
+
+## Purpose
+1. Provide a way to enact change in Canopy's frontend that impacts all projects and squads.
+2. Clarify what things we are aligned on across the entire front-end, so that everything else can be decided by the members of each squad.
+
+## What this isn't
+1. An infringement on the autonomy of squads. The idea is that if we're aligned on the few important things, we can be autonomous on everything else.
+2. Something oppressively prescriptive.
+3. A technical review process where people get feedback on difficult problems they're facing. Although important, the RFC is not how
+   technical reviews happen.
+
+## Signs that this is working
+1. The content is up-to-date and people care about it.
+2. There are regular updates.
+3. Things are regularly removed.
+4. We change the process when it's annoying or doesn't work well.
+5. Ideas have a safe place to develop before being criticized.
+
+## Signs that this isn't working
+1. This blocks people from getting stuff done.
+2. Ivory tower gatekeeping.
+3. Changes are discouraged instead of encouraged.
+4. "Well that's just the way it is."
+5. The process is heavy, bureaucratic, or long.
```

Comment:
I'd add to this list something like, "Some _other_ process is used."

Reply:
👍 will add

Reply:
Also maybe mention a distinction between this and the backlog

Reply:
👍 will add the technical backlog to the "what this isn't" section